api/models/likeDao.js

```diff
@@ -0,0 +1,41 @@
+const { AppDataSource } = require('./data_source');
+
+const getLikeUserId = async (userId, productId) => {
+    const result = await AppDataSource.query(
+        `
+		SELECT 
+           likes.user_id AS userId,
+           likes.product_id AS prodcutId
+		FROM likes
+		WHERE user_id =? AND product_id = ?`,
+        [userId, productId]
+    );
+    return result[0];
+};
+
+const createLike = async (userId, productId) => {
+    await AppDataSource.query(
+        `INSERT INTO likes(
+                user_id, 
+                product_id
+            ) VALUES (?, ?);
+            `,
+        [userId, productId]
+    );
+    const result = await getLikeUserId(userId, productId);
+
+    return result;
```

Comment:
```suggestion
    return await getLikeUserId(userId, productId);
```
코드는 가독성을 해치지 않는다면 한 줄이라도 줄일수록 좋습니다 :)